custom_components/plugwise/const.py

```diff
@@ -73,7 +81,50 @@
 UPPER_BOUND: Final = "upper_bound"
 
 # Sensor constants
+DHW_TEMP: Final = "dhw_temperature"
+DHW_SETPOINT: Final = "domestic_hot_water_setpoint"
+EL_CONSUMED: Final = "electricity_consumed"
+EL_CONSUMED_INTERVAL: Final = "electricity_consumed_interval"
+EL_CONSUMED_PEAK_INTERVAL: Final = "electricity_consumed_peak_interval"
+EL_CONSUMED_OFF_PEAK_INTERVAL: Final = "electricity_consumed_off_peak_interval"
+EL_CONSUMED_PEAK_CUMULATIVE: Final = "electricity_consumed_peak_cumulative"
+EL_CONSUMED_PEAK_POINT: Final = "electricity_consumed_peak_point"
```

Comment:
If we are shortening I would suggest improving this (pretty long names to pretty long names seems to defeat the point): `E_CONS_INT, E_CONS_PEAK, E_CONS_NPEAK`? Not sure, but lets make 'm shorter.

And just PH1/PH2/PH3 would do I guess

Reply:
I'll look at this later, thanks.

Reply:
And implemented.

Reply:
I didn't short the last word of the EL-related constants, when shorted they made less sense.